src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine caterpillar = new Excavator();
+          caterpillar.doWork();
```

Comment:
```suggestion
        Machine caterpillar = new Excavator();
        caterpillar.doWork();
```
тут не потрібні більші відступи, в чекстайлі ця помилка теж присутня